xml/System.Text.Json.Serialization/JsonIncludeAttribute.xml

```diff
@@ -29,11 +29,11 @@
 
 ## Remarks
 
-When applied to a property, indicates that non-public getters and setters can be used for serialization and deserialization. Non-public properties are not supported.
+When applied to a property, indicates that non-public *getters* and *setters* can be used for serialization and deserialization. In .NET 6 and .NET 7, non-public properties aren't supported.
```

Comment:
It might make sense to add a section clarifying that the source generator is limited to non-public members that are still visible to generated code (i.e. `internal` works fine if in the same project, `private` is mostly out of reach modulo exceptions).

Reply:
Assuming I understand Erik's comment correctly, the phrase we use in the C# reference is that "the property is accessible". Its use includes `public`, `internal` when in the same assembly, `protected` for derived classes, and the combos for `protected internal` and `private protected`.

Reply:
Exactly that. In some cases even `private` might work (for types that are nested inside the same class as the generated context) but that's splitting hairs. The term that Roslyn uses for that concept is ["accessible"](https://learn.microsoft.com/en-us/dotnet/api/microsoft.codeanalysis.compilation.issymbolaccessiblewithin?view=roslyn-dotnet-4.6.0) so perhaps we should use that here.

Reply:
@eiriktsarpalis Please check the latest updates. Not sure if I got it entirely correct.

Reply:
Looks good to me, thanks.